core/pom.xml

```diff
@@ -102,5 +102,10 @@
 			<artifactId>postgresql</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>org.apache.maven</groupId>
+			<artifactId>maven-model</artifactId>
+			<version>3.8.6</version>
```

Comment:
you can remove the version here because you set a `<dependencyManagement/>` in the project parent.

```suggestion
```